packages/expo-gl/src/index.ts

```diff
@@ -0,0 +1,4 @@
+import { GLView } from './GLView';
+export * from './GLView';
+export * from './GLView.types';
+export default GLView;
```

Comment:
Thinking we maybe should remove this so that it's consistently imported as `import { GLView } from 'expo-gl'`. Right now it's exported twice.